dist/google-music.js

```diff
@@ -452,7 +466,7 @@ proto.hooks = {
         //   <div id="playerSongInfo" style=""></div>
         //   <paper-icon-button icon="remove-circle-outline" data-rating="0" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0"></paper-icon-button>
         // jscs:enable maximumLineLength
-        if (target.dataset.rating !== undefined && target.hasAttribute('aria-label') &&
+        if (target.dataset !== undefined && target.dataset.rating !== undefined && target.hasAttribute('aria-label') &&
```

Comment:
Instead of doing an `undefined` check, we can prob loosen it up to a truthy check:

```js
if (target.dataset &&
```